### PR TITLE
Docs: Add documentation with regards to the new prettify function

### PR DESCRIPTION
**Pull request by olbo98**
**What this PR does / why we need it**:
This PR adds a short paragraph regarding the new prettify function that was added in #36324. Please comment if there's anywhere else in the docs where this information should be added.

### CODE REVIEW

**Comment by ivanahuckova** [comment]
@olbo98 There is a separate documentation for Logs Panel as well. 🙂

**Comment by olbo98** [comment]
> @olbo98 There is a separate documentation for Logs Panel as well. 🙂

Thanks @ivanahuckova i'll go ahead and change that as well! 👍

File: docs/sources/explore/logs-integration.md [hunk 1]
```diff
@@ -71,6 +71,10 @@ Shows or hides the unique labels column that includes only non-common labels. Al
 
 Set this to True if you want the display to use line wrapping. If set to False, it will result in horizontal scrolling.
 
+#### Prettify JSON
+
+Set this to true if you want to pretty print all JSON logs. This will not affect logs in any format other than JSON.
```
Comment (achatterjee-grafana) [inline docs/sources/explore/logs-integration.md:76]:
```suggestion
Set this to `true` to pretty print all JSON logs. This setting does not affect logs in any format other than JSON.
```

File: docs/sources/panels/visualizations/logs-panel.md [hunk 2]
```diff
@@ -35,5 +35,6 @@ Use these settings to refine your visualization:
 - **Unique labels -** Show or hide the unique labels column, which shows only non-common labels.
 - **Common labels -** Show or hide the common labels.
 - **Wrap lines -** Toggle line wrapping.
+- **Prettify JSON -** Set this to true if you want to pretty print all JSON logs. This will not affect logs in any format other than JSON.
```
Comment (achatterjee-grafana) [inline docs/sources/panels/visualizations/logs-panel.md:38]:
```suggestion
- **Prettify JSON -** Set this to `true` to pretty print all JSON logs. This setting does not affect logs in any format other than JSON.
```

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
Added review comments.

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

**Pull request merged**